apps/gkyl_gyrokinetic_priv.h

```diff
@@ -176,6 +176,8 @@ struct gk_lbo_collisions {
   struct gkyl_array *nu_sum_host, *prim_moms_host, *nu_prim_moms_host; // LBO primitive moments host-side for I/O
   bool normNu; // Boolean to determine if using Spitzer value
   struct gkyl_array *norm_nu; // Array for normalization factor computed from Spitzer updater n/sqrt(2 vt^2)^3
+  double *norm_nu_facs; // // List of collision freqs without factor of n_r/(v_ts^2+v_tr^2)^(3/2)
```

Comment:
Why is this a pointer? There seems to be an inconsistency here in how this normalization factor is defined between the public and private headers.